test/util/client.go

```diff
@@ -224,6 +224,24 @@ func WaitForClusterResourceQuotaCRDAvailable(clusterAdminClientConfig *rest.Conf
 	return nil
 }
 
+func WaitForSecurityContextConstraintCRDAvailable(clusterAdminClientConfig *rest.Config) error {
```

Comment:
Wouldn't it be better to make a common sub-function (accepting `schema.GroupVersionResource`) than just copypasting the same code for everything?

Reply:
yes, thanks